packages/react-components/react-toast/src/state/useToaster.ts

```diff
@@ -67,13 +119,31 @@ export function useToaster<TElement extends HTMLElement>(options: ToasterProps =
     const cleanupDismissListener = addToastListener(EVENTS.dismiss, dismissToast);
     const cleanupDismissAllListener = addToastListener(EVENTS.dismissAll, dismissAllToasts);
 
+    const focusShortcutListener = (e: KeyboardEvent) => {
+      if (isFocusShortcut(e)) {
+        pauseAllToasts();
+        const mostRecentToast = getMostRecentToast();
```

Comment:
```suggestion
        const mostRecentToast = getMostRecentVisibleToast();
```

nit This seems important